src/watch.js

```diff
@@ -339,6 +295,52 @@ class Watch {
     );
   }
 
+  /**
+   * Determines whether an error is considered permanent and should not be
+   * retried. Errors that don't provide a GRPC error code are always considered
+   * transient in this context.
+   *
+   * @private
+   * @param {Error} error An error object.
+   * @return {boolean} Whether the error is permanent.
+   */
+  isPermanentError(error) {
```

Comment:
function?

Reply:
These are members of the Watch class and hence don't need a `function` keyword.